apm-sniffer/apm-sdk-plugin/motan-plugin/src/main/java/org/apache/skywalking/apm/plugin/motan/define/MotanConsumerInstrumentation.java

```diff
@@ -41,7 +40,7 @@
      */
     private static final String CONSTRUCTOR_INTERCEPT_CLASS = "org.apache.skywalking.apm.plugin.motan.MotanConsumerInterceptor";
     /**
-     * Class that intercept {@link com.weibo.api.motan.rpc.AbstractProvider#call(Request)}.
+     * Class that intercept {@link com.weibo.api.motan.rpc.AbstractProvider#call(com.weibo.api.motan.rpc.Request)}.
```

Comment:
Don't use @link, use iteral string only.

Reply:
> Don't use @link, use iteral string only.

Use `{@code ..}` is also acceptable?

Reply:
I think both are safe for class loader. I recommend remove link because this class wouldn't exist in agent.